### PR TITLE
Fail PR if Warnings are present in React Code

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/QueryMultiSelector/QueryMultiSelector.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ const QueryMultiSelector = ({
   // Use effect to get the initial values from props
   useEffect(() => {
     if (
-      currentKeys.length === 1 &&
+      currentKeys.length == 1 &&
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Common/FormComponents/QueryMultiSelector/QueryMultiSelector.tsx:89]: i believe this is used for testing this PR and can be be reverted.

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Common/FormComponents/QueryMultiSelector/QueryMultiSelector.tsx:89]: yes! trying to get this to fail the PR :P

File: .github/workflows/react-no-warnings.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules
+        working-directory: ./portal-ui
+        run: yarn
+      - name: Check for Warnings in build output
+        working-directory: ./portal-ui
+        continue-on-error: false
+        run: |
+          if yarn build | grep "Compiled with warnings"; then echo "There are warnings in the code" && exit 1 ; fi
```
Comment (harshavardhana) [inline .github/workflows/react-no-warnings.yml:17]: You can use `./check-warnings.sh`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: .github/workflows/react-no-warnings.yml (hunk 2 above)
Comment (harshavardhana) [inline .github/workflows/react-no-warnings.yml:17]:
```suggestion
          chmod +x check-warnings.sh  && ./check-warnings.sh 
```

File: portal-ui/check-warnings.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if yarn build | grep "Compiled with warnings"; then
+  echo "There are warnings in the code"
+  exit 1
+fi
```
Comment (harshavardhana) [inline portal-ui/check-warnings.sh:6]:
```suggestion
fi

```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: portal-ui/Makefile [hunk 4]
```diff
@@ -4,3 +4,5 @@ build-static:
 	@echo "Building frontend static assets to 'build'"
 	NODE_OPTIONS=--openssl-legacy-provider yarn build
 
+test-warnings:
+	./check-warnings.sh
```
Comment (harshavardhana) [inline portal-ui/Makefile:8]:
```suggestion
	./check-warnings.sh

```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]
LGTM.

**Review by Alevsk — APPROVED** [review]
:+1:

**Pull request merged**